src/PrestaShopBundle/Controller/Admin/PrestaShopAdminController.php

```diff
@@ -72,12 +106,124 @@ protected function trans(string $id, array $parameters = [], ?string $domain = n
         return $this->container->get(TranslatorInterface::class)->trans($id, $parameters, $domain, $locale);
     }
 
+    protected function generateSidebarLink(string $section, ?string $title = null): string
+    {
+        if (empty($title)) {
+            $title = $this->trans('Help', [], 'Admin.Global');
+        }
+
+        $iso = $this->container->get(LanguageContext::class)->getIsoCode();
+        $url = $this->generateUrl('admin_common_sidebar', [
+            'url' => $this->container->get(Documentation::class)->generateLink($section, $iso),
+            'title' => $title,
+        ]);
+
+        // this line is allow to revert a new behaviour introduce in sf 5.4 which break the result we used to have
+        return strtr($url, ['%2F' => '%252F']);
+    }
+
+    /**
+     * Get error by exception from given messages
+     *
+     * @param array<string, string|array<int, string>> $messages
+     *
+     * @return string
+     */
+    protected function getErrorMessageForException(Throwable $e, array $messages): string
```

Comment:
This also comes from the `FrameworkBundleAdminController`, since this controller is meant to replace this class on all our controllers ultimately I preferred to duplicate the code and keep them completely independent so that can be reworked/refactored however we need

I could introduce a common trait but it would couple both base classes I'm not sure if it's interesting